Helpjuice/ckeditor5
# [Editor] [sc-2377] The Remove Format feature is not removing all types of extra formatting

## Describe the changes in this pull request

This PR changes the schema to allow `htmlAttributes` to be removed using the **Remove Format** feature.

App PR: https://github.com/hajrice/helpjuice4/pull/3099

### Demo:

#### Before

https://github.com/Helpjuice/ckeditor5/assets/28270050/c0541047-3bd2-46b1-b0f6-2c023b97619d

#### After

https://github.com/Helpjuice/ckeditor5/assets/28270050/24724eed-e83e-4bba-afa4-e6bd21a78c4e

### Does this resolve an existing issue?

[[sc-2377](https://app.shortcut.com/helpjuice/story/2377/the-remove-format-feature-is-not-removing-all-types-of-extra-formatting)]

### How should this be tested during the review?

Insert the following HTML through the source editor on the editor preview (I got this HTML from copying the content from this [file](https://github.com/Helpjuice/ckeditor5/files/12030630/Test.docx) and pasting with formatting in the editor; I just removed some unnecessary spacing and tags):

```
<h1>
    Test
</h1>
<h2>
    Section 1
</h2>
<p class="MsoNormal" style="text-indent:.5in;">
    A paragraph with some random thoughts.
</p>
<h2>
    Section 2
</h2>
<p class="MsoNormal">
    Another paragraph
</p>
```

Now, select everything and click on **Remove Format** (the eraser icon in the toolbar). It will remove all the custom classes and styles (you can check the source editor to verify this).
